src/queries/tags/update.js

```diff
@@ -0,0 +1,40 @@
+import db from '../../models';
+
+/**
+ * @param {object} newTags tag lists data to be saved in db
+ * @param {object} slug article identify
+ * @param {object} action action method to perform¬
+ * @returns {object} Array of updated tag list of the article
+ */
+export default async (newTags, slug, action) => {
+  let result = [];
+  let message = '';
+  const response = await db.Article.findAll({
+    limit: 1,
+    where: { slug },
+    attributes: ['tagList'],
+    logging: false
+  });
```

Comment:
I can see that you find an article by slug not by id, so why

Reply:
As it is mentioned in the app **README** and **PT Stories**,  the identifier of the article is `slug` not `id`

Reply:
I think that has been done.. you did well to create another file